core/src/main/resources/lib/ophan.fjs

```diff
@@ -56,9 +56,8 @@ export const MediaEvents: { [MediaEvent]: MediaEvent } =
   { REQUEST: 'REQUEST'
   , READY: 'READY'
   , PLAY: 'PLAY'
-  , PERCENT25: 'PERCENT25'
-  , PERCENT50: 'PERCENT50'
-  , PERCENT75: 'PERCENT75'
-  , THE_END: 'THE_END'
+  , PERCENT25: '25'
```

Comment:
The keys are of type `MediaEvent`, meaning they must be translated too.

Reply:
As discussed this morning (!) I re-tested locally and it looks ok.